README.md

```diff
@@ -2,4 +2,159 @@
 
 ## Objectives
 
-## Instructions
\ No newline at end of file
+1. Create custom data models in Swift:
+  * Declare properties.
+  * Write initializers.
+  * Write methods.
+2. Use the `CustomDebugStringConvertible` protocol to determine how the `print()` function represents your classes in the debug console.
+3. Declare properties as readonly.
+4. Declare properties as private.
+5. Write and utilize private methods.
+6. Write a custom setter.
+7. Use calculated properties.
+8. Subclass in Swift:
+  * Override the superclass's initializer.
+  * Extend the class.
+9. Combine the usage of your classes to assemble a working back-end.
+
+## Introduction
+
+In this exercise, we're going to write a new implementation of Blackjack that uses Swift. You'll notice that a lot of the underlying thought structure remains the same, however Swift has some different features that makes a slightly different approach to the problem more appealing. 
+
+### Twenty-One
+
+[Blackjack](https://en.wikipedia.org/wiki/Blackjack), also known as "twenty-one", is a casino card game in which a player attempts to build a hand that is closer to a score of 21 than the house (or dealer's) hand without going over a total of 21 (or "busting"). The Ace's base value is one (1), but can be used as a value of eleven (11) if doing so does not bust a hand. Players are initially dealt two cards and choose to be dealt additional cards (to "hit") or to hold their hand until the end of the round (to "stay").
+
+## Instructions
+
+Open `swift-blackjack.xcworkspace`. 
+
+You may deselect the target membership of the testing files in order to work on the classes-one-by-one. This option is found in the File Inspector pane of the Utilities area when the relevant file is selected in the Project Navigator pane of the Navigator area.  
+**Note:** *Doing this may report false-positives to your Learn.co profile's local build light, so make sure you have test results for all five custom classes before regarding your solution as complete.*
+
+### `Card.swift`
+
+1. Re-enable the target membership for `CardSpec.swift` for `swift-blackjackTests`.
+
+2. Create a class file for the `Card` class. It will need the same properties as the `FISCard` class from `objc-blackjack`:
+  * `suit`, a string,
+  * `rank`, a string,
+  * `cardLabel`, a string, and
+  * `cardValue`, an unsigned integer.
+
+3. Write an initializer for `Card` that takes two arguments for `suit` and `rank`.
+  * It should also set the `cardLabel` property to string containing both the suit and the rank in the form of `♠︎A` (for the Ace of Spades).
+  * It should also set the `cardValue` property appropriately for scoring Blackjack: Aces are worth one, number cards are worth their face values, and face cards are worth ten.
+
+4. Make the `Card` class conform to the `CustomDebugStringConvertible` protocol. A `Card` object should print its `cardLabel` string to the console when it is submitted to the `print()` function.
```

Comment:
`print()` is a function http://stackoverflow.com/questions/25951195/swift-print-vs-println-vs-nslog and the `CustomDebugStringConvertible` protocol is the Swift version of overriding the `description` method in Objective-C. Since Swift base classes don't inherit from NSObject, they don't come with the property/method and from what I found online must conform to this protocol which adds a single string property called `debugDescription`. I can't think of a way to avoid doing this for the console version of this lab. As protocols go, this is probably the simplest one. I could add a quick section before the instructions explaining it?